benchmarks/partitioned_dataset_filter_benchmark.py

```diff
@@ -0,0 +1,64 @@
+import conbench.runner
+
+from benchmarks import _benchmark
+
+
+@conbench.runner.register_benchmark
+class PartitionedDatasetFilterBenchmark(_benchmark.Benchmark, _benchmark.BenchmarkR):
+    """
+    Read and filter a partitioned dataset.
+
+        PartitionedDatasetFilterBenchmark().run(<case>, options...)
+
+    Parameters
+    ----------
+    case : str
```

Comment:
Case isn't a string.

```
    case : sequence, optional
        Benchmark options as a sequence (rather than individual params):
        [<query>]
```

Reply:
Missing `query`.

```
    query : string, optional
        The filter case (one of: "vignette", "payment_type_3",
        "small_no_files")
```

Reply:
Also missing (which should be listed first):

```
    all : boolean, optional
        Run all benchmark cases
```

Reply:
Actually, I don't want new contributors to have to struggle with doc strings.

One short sentence would be perfect instead. Pattern to follow:

https://github.com/ursacomputing/benchmarks/commit/51795a75a2581e7a00a815263667438ef3db9037